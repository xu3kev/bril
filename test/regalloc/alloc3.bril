../reginterp/alloc3.bril