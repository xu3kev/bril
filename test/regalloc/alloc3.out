main {
	vj: int = const 1;
	r_03: int = const 2;
	r_01: int = const 1;
	r_04: int = const 2;
	v3: int = const 3;
	v0: int = const 5;
	r_02: int = add vj r_01;
	r_01: int = sub r_03 v0;
	r_02: int = mul r_02 r_01;
	r_01: int = add vj r_04;
	r_03: int = add vj v3;
	r_02: int = id r_02;
	r_01: int = add r_01 v0;
	r_01: int = id r_01;
	r_03: int = add r_03 v0;
	vj: int = id r_02;
	print vj;
	print r_03;
	print r_01;
}
